### PR TITLE
Add logic to calculator with Arithmetic operations

### PR DESCRIPTION
**Pull request by Atifzada**
### The following changes have been made to this PR.

- Created a new directory called logic inside your project
- Copied the files calculate.js and operate.js into your logic/ directory
- Installed [big.js](https://www.npmjs.com/package/big.js) using NPM
- Imported the files into your Calculator.js component
- Implemented click event handlers using the math logic from calculate.js and operate.js in your React component
- The goal achieved to make all calculator buttons work according to their functionality e.g. the - button should subtract a given value
- Tested app in the browser. The Calculator is performing math operations.
- Checked for linter errors
- Updated Readme file

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,10 +1,55 @@
...
+        {calcState.result}
+        {calcState.function}
+        {calcState.next}
+      </div>
+      <div className="btnsContainer">
+        <div>
+          <button type="button" className="Btns" onClick={handleClick}>AC</button>
+          <button type="button" className="Btns" onClick={handleClick}>+/-</button>
+          <button type="button" className="Btns" onClick={handleClick}>%</button>
+          <button type="button" className="Btns MathOperations">÷</button>
```
Comment (roshan-bajgain) [inline src/components/Calculator.js]: - [x] I notice that you did not implement the click events on `÷` so it is not working please implement it and ask for another reviewer.
` <button type="button" className="Btns MathOperations" onClick={handleClick}>÷</button>`

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Atifzada, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (Atifzada) [reply src/components/Calculator.js]: Thank you so very much for pointing out the error. I will come up with solution

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Atifzada 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Well done on making the requested changes by the previous reviewer
- Excellent communication

Cheers, and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**